### PR TITLE
[ARGO-256] Fixes for status reponses

### PR DESCRIPTION
**Pull request by makistsantekidis**
### Description
Current status requests do not use the latest error responses and marshalling.

### Implementation
Align all the status responses to the current schema

### CODE REVIEW

**Comment by makistsantekidis** [comment]
please review @kaggis @pkoro

**Comment by makistsantekidis** [comment]
please review @dpavlos @kaggis @pkoro

File: respond/validators.go [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+func validateDate(dateStr string) (int, error) {
+	parsedTime, err := time.Parse(zuluForm, dateStr)
+	if err != nil {
+		return 0, err
+	}
+	ymdTime, err := strconv.Atoi(parsedTime.Format(ymdForm))
+	return ymdTime, err
+}
+
+func VadlidateDateRange(dateStart string, dateEnd string) (int, int, []ErrorResponse) {
```
Comment (dpavlos) [inline respond/validators.go]: Perhaps the function's name should be ``ValidateDateRange``? Of course should also be changed in all references above.

**Comment by kaggis** [comment]
+1

**Pull request merged**